### ISSUE TITLE
a problem 

### ISSUE THREAD

**Issue by IamMegatron2025** [MCGallaspy/cascade-trainer#1]
hello, I like this work. But it seems to have a problem. Should we add a line of code:
`cmd=' '.join(cmd)` after [this place](https://github.com/MCGallaspy/cascade-trainer/blob/master/annotate.py#L90)?

**Comment by MCGallaspy** [comment]
I don't think so. According to the [subprocess docs](https://docs.python.org/3/library/subprocess.html#frequently-used-arguments):

> args is required for all calls and should be a string, or a sequence of program arguments. Providing a sequence of arguments is generally preferred, as it allows the module to take care of any required escaping and quoting of arguments (e.g. to permit spaces in file names).

**Comment by IamMegatron2025** [comment]
@MCGallaspy I use ubuntu. I add it and it works
